packages/sdk/src/services/credential-service.ts

```diff
@@ -22,8 +21,21 @@ export interface GenerateCredentialExtendedDto extends Omit<GenerateCredentialDt
   issuer: string | { id: string; name?: string };
 }
 
+export interface CredentialDetail {
+  issuer: any | null;
+  subject: string | null;
+  issuanceDate: Date | null;
+  expirationDate: Date | null;
+  id: string | null;
+  credentialSchema: any | null;
```

Comment:
@vijayshiyani this should not be just `any`. When assigning value to the slot, it expects object with an `id` property: `{id: string}`;

Reply:
@linasi fix typings 
```
export interface CredentialDetail {
  issuer?: string | { id: string; name?: string };
  subject?: string;
  issuanceDate?: Date;
  expirationDate?: Date;
  id?: string;
  credentialSchema?: { id: string; type: string };
  revocable?: boolean;
}
```